### ISSUE TITLE
Data types mapping

### ISSUE THREAD

**Issue by ElMaxo** [k2informatics/erloci#10]
Please, add data types mapping (Oracle <--> Erlang) to documentation for reference.

**Comment by c-bik** [comment]
https://github.com/k2informatics/erloci/commit/c9724162f2ab87e5b4f4f965af72e3efaa9e99f1

Can you please tell me if this is any useful to you?

**Comment by ElMaxo** [comment]
Yes, this information is very useful. I'll try to add some info too today. I have some info about SQLT_NUM column and nuances with it.
